### PR TITLE
Add droput and its derivative in Keras converter

### PR DESCRIPTION
**Pull request by wenbingl**
Support Dropout in Keras. This can address #124.

### CODE REVIEW

File: onnxmltools/convert/keras/operator_converters/Dropout.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from keras.layers import Dropout, SpatialDropout1D, SpatialDropout2D, SpatialDropout3D
+
+from ...common._registration import register_converter
+from ...common._apply_operation import apply_identity
```
Comment (wschin) [inline onnxmltools/convert/keras/operator_converters/Dropout.py]: Please swap`*_reg*` and `*_apply*`. Thanks.

File: onnxmltools/convert/keras/operator_converters/__init__.py [hunk 2]
```diff
@@ -26,4 +26,4 @@
 from . import SimpleRNN
 from . import Upsample
 from . import ZeroPad
-
+from . import Dropout
```
Comment (wschin) [inline onnxmltools/convert/keras/operator_converters/__init__.py]: Move it right after `from . import Dot` please.

File: tests/end2end/test_single_operator_with_cntk_backend.py [hunk 3]
```diff
@@ -162,12 +162,15 @@ def test_dense(self):
...
+        hidden = Dense(D, activation='relu')(input)
+        result = Dropout(0.2)(hidden)
+
+        keras_model = Model(inputs=input, outputs=result)
         keras_model.compile(optimizer='adagrad', loss='mse')
 
-        coreml_model = coremltools.converters.keras.convert(keras_model)
-        onnx_model = onnxmltools.convert_coreml(coreml_model)
+        # coremltool cannot convert this kind of model.
+        # coreml_model = coremltools.converters.keras.convert(keras_model)
```
Comment (wschin) [inline tests/end2end/test_single_operator_with_cntk_backend.py]: If so, please create another tests for `Dropout`. If Core ML is not able to handle it, you can call `_test_one_to_one_operator_core_keras`.

File: onnxmltools/convert/keras/operator_converters/Dropout.py (hunk 1 above)
Comment (wenbingl) [reply onnxmltools/convert/keras/operator_converters/Dropout.py]: switched

File: onnxmltools/convert/keras/operator_converters/__init__.py (hunk 2 above)
Comment (wenbingl) [reply onnxmltools/convert/keras/operator_converters/__init__.py]: switched

File: tests/end2end/test_single_operator_with_cntk_backend.py (hunk 3 above)
Comment (wenbingl) [reply tests/end2end/test_single_operator_with_cntk_backend.py]: added a new one.
